v2/gcs-to-sourcedb/src/test/java/com/google/cloud/teleport/v2/templates/GCSToSourceDbWithoutReaderIT.java

```diff
@@ -191,4 +292,21 @@ private void createAndUploadShardConfigToGcs(
     LOG.info("Shard file contents: {}", shardFileContents);
     gcsResourceManager.createArtifact("input/shard.json", shardFileContents);
   }
+
+  public void createAndUploadJarToGcs(GcsResourceManager gcsResourceManager)
+      throws IOException, InterruptedException {
+    String[] shellCommand = {"/bin/bash", "-c", "cd ../spanner-custom-shard"};
```

Comment:
I'm surprised that this is working. I thought the working directory would be outside v2 directory. Are you sure that this IT is getting executed in the github action? Can you make sure that this doesn't break if executed from the outermost directory (outside v2/)

Reply:
Yup its working